src/multibody/joint/joint-planar.hpp

```diff
@@ -412,30 +415,34 @@ namespace se3
         // else    : S = -Sp
         // S / t = Sp / |t|
         // S * S = - I2
-        // R = I2 + ( 1 - ct) / |t| * S + ( 1 - st / t ) * S * S
-        //   =      ( 1 - ct) / |t| * S +       st / t   * I2
+        // R = I2 + ( 1 - ct) / |t| * S + ( 1 - st / |t| ) * S * S
+        //   =      ( 1 - ct) / |t| * S +       st / |t|   * I2
         //
         // Ru = exp3 (w)
         // tu = R * v = (1 - ct) / |t| * S * v + st / t * v
         //
         // M0 * Mu = ( R0 * Ru, R0 * tu + t0 )
 
         Eigen::VectorXd::ConstFixedSegmentReturnType<2>::Type v = vs.segment<2>(idx_v());
-        double ct,st; SINCOS (theta, &st, &ct);
+        Eigen::Matrix<Scalar,2,1> cst;
```

Comment:
For `res.tail<2>() = R0 * cst;` which makes more sense to me than writing trigonometric relations (`cst` is a element of R^2 that we rotate with `R0`)